Packs/MicrosoftCloudAppSecurity/Classifiers/classifier-MicrosoftCloudAppSecurity_5_9_9.json

```diff
@@ -0,0 +1,52 @@
+{
+	"id": "575cb861-e842-4e01-8379-1c9f55bd3b38",
```

Comment:
update the id to be `MicrosoftCAS`

Reply:
Actually, let's fix the IDs everywhere.
If the `format` command doesn't do that, let's open a dev bug